### ISSUE TITLE
Can i use script-message to show and give the display seconds?

### ISSUE THREAD

**Issue by kesdoputr** [donmaiq/Mpv-Playlistmanager#31]
Hello, i often use toggle mode so i set
**open_toggles = true
playlist_display_timeout = 86400**

Now i want to call it from other script and I use below command to show the playlist
**mp.commandv("script-message", "playlistmanager", "show", "playlist")**
But because i set a long timeout the playlist will not disappear automatic

Is it possible to set the display seconds without change the setting?
ex.I can use script-message to call it for 3sec display, and i still can use the original hotkey to call it in toggle mode.

Thanks for your watching.

**Comment by jonniek** [comment]
Done 9f9e067 

```
mp.commandv("script-message", "playlistmanager", "show", "playlist", 2)
```

Let me know if it works for you. Right now it only changes the timeout for the first render of playlist. That means that if you interact with the playlist or it renders again from some event then it will change to the default timeout. This can be changed if it doesn't suit your usecase.

**Comment by kesdoputr** [comment]
Thanks for your quickly reply. I try but it seems not work.
If i set playlist_display_timeout = 5 and use 
mp.commandv("script-message", "playlistmanager", "show", "playlist", 1)
Playlist will show for 5 secs not 1sec.

I use it on the betterchapters.lua
https://github.com/Argon-/mpv-config/blob/master/scripts/betterchapters.lua

And i want to change the **mp.commandv("script-message", "osc-playlist")** to your playlistmanager
So i need to show playlist temporary for a timeout.
Thanks for your watching.

**Comment by kesdoputr** [comment]
Add more test results.
As i say i use in betterchapters.lua that it will show playlist when change to next or previous file.
And i found that the code works if it's only one file, will show the playlist for 1sec.
`mp.commandv("script-message", "playlistmanager", "show", "playlist", 1)`
But if more then one file, when change to next or previous file will show the playlist for the timeout in setting.

PS.I try to use in keybind that works.
**Ctrl+u script-message playlistmanager show playlist 1**

**Comment by kesdoputr** [comment]
Sorry for more test, **it seems not work when it occur on file change.**
ex. 
There ar two files in playlist.
With **betterchapters.lua**, when i press the previous chapter hotkey at the first chapter of first file
it will show the playlist for the secs i give in script-message.
But when i press the next chapter hotkey at the last chapter of first file
it will go to the second file and show the playlist for the timeout in config file

The same results, when i press the previous chapter hotkey at the first chapter of second file
it will go to the first file and show the playlist for the timeout in config file

Thanks for your watching.

**Comment by jonniek** [comment]
Fix 84cec7e9ad680276c401cdeb28c2c3193de9e5b4

The timeout should now be 1 second only until playlist disappears. Let me know if there's any other issues.

**Comment by kesdoputr** [comment]
I test and it's work perfect, thanks for your quickly reply
